src/Resources/skeleton/stimulus/Controller.tpl.php

```diff
@@ -0,0 +1,7 @@
+import { Controller } from '@hotwired/stimulus';
+
+export default class extends Controller {
+    connect() {
```

Comment:
```js
    // static targets = [ 'fields', 'field', 'addButton' ]
    // static values = {
    //    prototype: String,
    //    maxItems: Number,
    //    itemsCount: Number,
    // }
```
maybe adding this in the template commented ?

Reply:
@zairigimad I used the example given in Stimulus bridge doc, to have a minimal template

Reply:
what about adding an interactive mode?
then defining this targets and values
and outputting a default html template at the end of the generation ?

it would be 🥇

Reply:
I'd say 👎 for adding commented out `targets` & `values` in the template. I'm open to having maker ask something along the lines of `Do you want to include targets?...` then adding them into the template if the user says yes.